www/vorj00/ukol_05/js/main.js

```diff
@@ -0,0 +1,180 @@
+        default:
+            return console.log("zadej prosím jednotku `C` nebo `F`");
+    }
+
+    console.log("Výsledek je:", result+"°"+resultUnit)
+}
+
+temperatureConverter(451, "C");
+temperatureConverter(451, "D");
+temperatureConverter(451, "F");
+temperatureConverter(20, "F");
+temperatureConverter(68, "C");
+
+getAge(2000);
+getAge(1999)
+getAge(2001);
+getAge(2048);
+
+/**
+ * %CENSORED%. Vytvořte funkci, která vezme 2 číselné argumenty a vrátí podíl prvního čísla a druhého čísla 
+ * v procentech. Výsledek vypište do konzole, např. 21 je 50% z 42. Pro zkrácení / zaokrouhlování desetinných 
+ * míst použijte funkci .toFixed(n). Např. var pi = 3.1415926535; pi.toFixed(2); Pozor na dělení nulou! 
+ */
+/* opět jiný zápis */
+var percentage = function(n1, n2){
+    if(n2 == 0){
+        return console.log("Dělení nulou!")
+    };
+
+    n1 = n1.toFixed(2);
+    n2 = n2.toFixed(2);
+
+    let percentage = (n1 / n2 * 100).toFixed(2);
+    console.log(n1, "je", percentage+"% z", n2);
+}
+
+percentage(21,42);
+percentage(42,21);
+percentage(0,21);
+percentage(0,0);
+percentage(21,0);
+percentage(pi,259);
+percentage(3.1415926535,259);
+
+/**
+ * Kdo s koho. Vytvořte funkci, která vezme 2 číselné argumenty a vrátí ten větší z nich. Pokud se čísla 
+ * rovnají, vypište, že se rovnají. Vyzkoušejte funkčnost pro celá čísla, desetinná čísla, zlomky. Zkuste 
+ * je párkrát zavolat v kódu a výsledky uložit do proměnných. 
+ */
+var maxNumber = (n1, n2) => {
+    if(n1 == n2){
+        console.log("Čísla se rovnají");
+    } else {
+        console.log("Větší je číslo",Math.max(n1, n2));
+    }
+}
+
+maxNumber(1,0);
+maxNumber(1,10);
+maxNumber(1,1);
+maxNumber(-1,0);
+
+// trololoJS
+maxNumber(1,"1");
+maxNumber(1,true);
+maxNumber(0,[]);
+maxNumber(9, (!+[]+[]+![]).length);
+maxNumber(91, 9+"1");
+
+/**
+ * I can cleary see the pattern. Vytvořte funkci, která vypíše popořadě všechny násobky 13, které jsou menší 
+ * nebo rovno 730. Použijte for loop. 
+ */
+var times13 = () => {
+    for(let i = 0; i <= 730; i+=13){
+        console.log(i);
+    }
+}
+
+times13();
+/**
+ * Around and about. Vytvořte funkci, která vypočte obsah kružnice podle dodaného poloměru. 
+ */
+var circleArea = radius => pi * radius**2;
+
+/**
+ * Another dimension. Vytvořte funkci, která vypočte objem kuželu, pokud znáte jeho výšku a poloměr. 
+ */
+var coneVolume = (height, radius) => 1/3 * pi * radius**2 * height;
+
+/** 
+ * Not sure if triangle, or just some random values. Vytvořte funkci, která rozhodne, zda se z 
+ * dodaných 3 délek dá postavit trojúhelník, tj. vypíše buď true/yes nebo false/no. 
+ */
+var isTriangle = (n1, n2, n3) => {
+    let result = true;
+    let values = [n1, n2, n3];
+    let sum = values.reduce((a, b) => a + b, 0); // sečtu
+
+    values.forEach((value) => {
+        if((sum - value) < value){ // vynechávám záměrně rovno, protože by pak hodnota byla 0 a nedávalo by to smysl
+            result = false;
+            return;
```

Comment:
tenhle return nezastavi cyklus foreach, musis throw, ale to nechces, takze dropback to for loop